pjmedia/include/pjmedia/port.h

```diff
@@ -563,6 +563,37 @@ PJ_DECL(pj_status_t) pjmedia_port_add_ref( pjmedia_port *port );
 PJ_DECL(pj_status_t) pjmedia_port_dec_ref( pjmedia_port *port );
 
 
+/**
+ * This is a helper function to add port destructor handler.
```

Comment:
It's not obvious when users can, or should, call this function? Should we call pjmedia_port_init_grp_lock() first? Or after adding the port to conf?

Reply:
After adding the port to conf. Updated #3928 desc issue 1 point ii. Should it be added to the function docs (also about avoiding premature pool release)?

Reply:
Yes, please.

Adding to the doc will be of great help in the future so we can easily recall how we're supposed to do this.